src/porepy/numerics/fv/mpfa.py

```diff
@@ -513,39 +522,74 @@ def _flux_discretization(
-        face continuity points, while C consists of +- 1 (depending on which side
-        the cell is relative to the face normal vector). The fourth equation
-        enforces the pressure to be unity in one cell at a time. Thus (i)-(iv) can
-        be inverted to express the pressure gradients in terms of the cell
-        center variables, that is, we can compute the basis functions on the
-        sub-cells. Because of the method construction (again see reference paper),
-        the basis function of a cell c will be non-zero on all sub-cells sharing
-        a vertex with c. Finally, the fluxes as functions of cell center values are
-        computed by insertion into Darcy's law (which is essentially half of A from
-        (i), that is, only consider contribution from one side of the face.
-        Boundary values can be incorporated with appropriate modifications -
-        Neumann conditions will have a non-zero right hand side for (i), while
-        Dirichlet gives a right hand side for (iii).
+        relating flux to the pressure. The third equation gives pressure continuity over
+        interior faces, thus B will contain distances between cell centers and the face
+        continuity points, while C consists of +- 1 (depending on which side the cell is
+        relative to the face normal vector). The fourth equation enforces the pressure
+        to be unity in one cell at a time. Thus (i)-(iv) can be inverted to express the
+        pressure gradients in terms of the cell center variables, that is, we can
+        compute the basis functions on the sub-cells. Because of the method construction
+        (again see reference paper), the basis function of a cell c will be non-zero on
+        all sub-cells sharing a vertex with c. Finally, the fluxes as functions of cell
+        center values are computed by insertion into Darcy's law (which is essentially
+        half of A from (i), that is, only consider contribution from one side of the
+        face. Boundary values can be incorporated with appropriate modifications -
+        Neumann conditions will have a non-zero right hand side for (i), while Dirichlet
+        gives a right hand side for (iii).
+
+        Parameters:
+            sd: The grid to be discretized. k: Permeability tensor. bnd: Boundary
+            condition for the problem. inverter: Which method to use for inverting the
+            local linear system in mpfa. ambient_dimension: Dimension of the full
+            computational domain. Defaults to
+                the dimension of ``sd``.
+            eta: Location of the Mpfa continuity point. If not provided, it will be set
+                to what is considered the best value for the given grid.
+
+        Returns:
+            A tuple of 6 elements:
+
+            :obj:`~scipy.sparse.spmatrix`: ``(shape=(sd.num_faces, sd.num_cells))``
+
+                Flux discretization matrix.
```

Comment:
Don't we need a blank line below each description?

Reply:
This is not the way @vlipovac does it, see e.g.,  https://github.com/pmgbergen/porepy/blob/ad271d465edf9aeff9745ea75e72a1c56473c0e5/src/porepy/geometry/map_geometry.py#LL324-L331C50. I am fine with always using a blank, though, if we can agree on that.

Reply:
A blank line *below* the indented content is not necessary, but above if there are specifications.
The next parameter entry will cause a linebreak automatically, but we need one for formatting reasons below specifications like shape.

Also, please drop the colon after " tuple of 6 elements", since Sphinx will interpret that half sentence as a type annotation otherwise.

Reply:
Will fix the colon.

Reply:
I put them there for readability in the py files, they help to recognize the whole text structure.
But it shouldn't make any difference for Sphinx, just a matter of taste.

On a very detailed notice, depending on the theme, the directive and some HTML/CSS scalings, a blank line might cause some minor vertical whitespace (like between paragraphs).

Very happy to have you guys behind these whole documentation topic so full-heartedly! 😀